prov/gni/include/gnix.h

```diff
@@ -293,7 +300,12 @@ struct gnix_ep_name {
 		uint32_t cm_nic_cdm_id : 24;
 		uint32_t cookie;
 	};
-	uint64_t reserved[4];
+	struct {
+		uint32_t rx_ctx_cnt : 8;
+		uint32_t unused1 : 24;
+		uint32_t unused2;
+	};
+	uint64_t reserved[3];
```

Comment:
This is an odd separation of unused/reserved (why isn't reserved still 4 words?), but maybe I'll understand this better as I review it.